sandbox/public/index.html

```diff
@@ -200,6 +200,59 @@
         clickCollectionEnabled: false,
       });
     </script>
+
+    <script nonce="%REACT_APP_NONCE%">
+      alloy("configure", {
+        clickCollectionEnabled: true,
+        clickCollection: {
+          internalLinkEnabled: true,
+          externalLinkEnabled: true,
+          downloadLinkEnabled: true,
+          sessionStorageEnabled: false,
+          eventGroupingEnabled: true,
+          filterClickDetails: (props) => {
+            console.log("filterClickDetails", props);
+          },
+        },
+        datastreamId: "76d1c882-2642-44ff-ad8e-ce32050d13d4",
+        orgId: "97D1F3F459CE0AD80A495CBE@AdobeOrg",
+        debugEnabled: true,
+        onBeforeLinkClickSend: function (options) {
```

Comment:
This is the new callback. It might make sense to include a commented out example from the public facing docs here.